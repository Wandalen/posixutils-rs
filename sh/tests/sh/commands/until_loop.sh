until true
do
    echo "inside loop"
    break
done
echo $?

until false
do
    echo "inside loop"
    break
done
echo $?