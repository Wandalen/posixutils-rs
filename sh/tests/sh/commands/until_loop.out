0
inside loop
0
